src/themes/ec/variables/_tag.scss

```diff
@@ -2,6 +2,7 @@
 @use '../index' as *;
 
 $tag: (
+  font: var(--f-ui-s),
```

Comment:
Did you check the result of this..? Because there is no font ui in EC. This is the value for EU, instead.